documentation/en/faqs.md

```diff
@@ -54,6 +54,11 @@ running `rm ~/.lotus ~/.lotusstorage`.
 Then, install Lotus afresh by following the instructions 
 found [here](https://docs.lotu.sh/en+getting-started).
 
+### Can I configure where the node's config and data goes?
+
+Yes! The `LOTUS_PATH` variable sets the path for where the Lotus node's data is written.
+The `LOTUS_STORAGE_PATH` variable does the same for miner-specific information.
```

Comment:
Will need to update one https://github.com/filecoin-project/lotus/pull/2317 goes in